style/style.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include "style/style.h"
 
+#include "util/string.h"
```

Comment:
This should be in the first commit

Reply:
Good catch! Switching the order of them was a last-minute change, so I messed up testing them. :x